### DISCUSSION TITLE
When in dark mode text input color is not set properly

### DISCUSSION THREAD

**Discussion by kylecotter** [trendyminds/craft-palette#22] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/505894/195959739-d62d8981-6b51-49a2-866a-ad85beadb90f.png)
![image](https://user-images.githubusercontent.com/505894/195959832-f9a754e3-440b-4425-9f92-5ea21df6034c.png)

When using in the CP the text input seems to inherit the default Craft grey color. When using on the frontend it inherits whatever the default input color is.

**Comment by aaronbushnell** [comment, marked as answer]
This is fixed for both Craft 4 (3.0.2) and Craft 3 (1.7.2). Thanks for reporting!
